From a5193b0eab8cb02452a3a0093800cde4322c104b Mon Sep 17 00:00:00 2001
From: Brittany Wald <brittany.wald@joyent.com>
Date: Thu, 7 Sep 2017 22:53:33 +0000
Subject: [PATCH] MANTA-3413 node-fash seems to intend add-data behavior it
 does not enable Reviewed by: Tim Kordas <tim.kordas@joyent.com> Reviewed by:
 Cody Peter Mello <cody.mello@joyent.com> Approved by: Kelly McLaughlin
 <kelly.mclaughlin@joyent.com>

---
 bin/fash.js  | 2 +-
 package.json | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/bin/fash.js b/bin/fash.js
index ea04d9c..607cc17 100755
--- a/bin/fash.js
+++ b/bin/fash.js
@@ -210,7 +210,7 @@ Fash.prototype.do_deserialize_ring.help = (
 Fash.prototype.do_add_data = function (subcmd, opts, args, callback) {
     var self = this;
 
-    if (opts.help || args.length !== 0 || !opts.v || !opts.d || !opts.b) {
+    if (opts.help || args.length !== 0 || !opts.v || !opts.b) {
         this.do_help('help', {}, [subcmd], function (err) {
             return callback(err ? err : true);
         });
diff --git a/package.json b/package.json
index 8935c96..4f08587 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "dynamo",
     "hash ring"
   ],
-  "version": "2.4.0",
+  "version": "2.4.1",
   "repository": {
     "type": "git",
     "url": "https://github.com/yunong/node-fash.git"
-- 
2.21.0

